protocol/client/messenger.go

```diff
@@ -256,13 +256,6 @@ func (m *Messenger) Send(c Contact, data []byte) error {
 	}
 
 	log.Printf("[Messenger::Send] sent message with hash %x", hash)
-
-	message.ID = hash
```

Comment:
Won't it be a problem for private chats? When a message encrypted with a public key is sent, it's not received by the sender from the Whisper network and that's why we had this code fragment.

Reply:
thanks, you right it will

Reply:
thanks, you right it will